packages/render-fidelity-tools/src/components/renderers/babylon-viewer.ts

```diff
@@ -0,0 +1,151 @@
+/* @license
+ * Copyright 2019 Google LLC. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License atQ
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import '@babylonjs/loaders/glTF';
+import {ArcRotateCamera, Engine, HDRCubeTexture, Scene, SceneLoader, Vector3} from '@babylonjs/core';
+import {css, customElement, html, LitElement, property} from 'lit-element';
+
+import {ScenarioConfig} from '../../common.js';
+
+// const IS_BINARY_RE = /\.glb$/;
```

Comment:
i don't know what does this variable do, i guess it's checking whether the model is glb or gltf? If that's the case I can just delete it because in Babylonjs you don't have to distinguish these two format

Reply:
Yes, that's right.

Reply:
I will delete this then.